<launch>
	
	<param name="global_example" value="global value" />
	
	<group ns="foo">
		<param name="utterance" value="Hello world!" />
		<param name="to_delete" value="Delete me!" />

		<group ns="gains">
			<param name="P" value="1.0" />
			<param name="I" value="2.0" />
			<param name="D" value="3.0" />
		</group>

		<node pkg="rospy_tutorials" name="param_talker" type="param_talker.py" output="screen">

			<param name="topic_name" value="chatter" />

		</node>

	</group>

</launch>
